kalite/distributed/settings.py

```diff
@@ -201,3 +201,34 @@ def USER_FACING_PORT():
 ########################
 
 LOCKDOWN = getattr(local_settings, "LOCKDOWN", False)
+
+
+########################
+# Screenshots
+########################
+
+from django.conf import settings
+PROJECT_PATH = os.path.realpath(getattr(local_settings, "PROJECT_PATH", settings.PROJECT_PATH)) + "/"
+
+SCREENSHOTS_OUTPUT_PATH = os.path.join(os.path.realpath(PROJECT_PATH), "..", "data", "screenshots")
+SCREENSHOTS_EXTENSION = ".png"
+
+SCREENSHOTS_DATABASE_NAME = "screenshot-data.sqlite"
```

Comment:
I think it's better to default to an in-memory database (by specifying `:memory:`) to make the database interactions faster, and make sure it's cleaned up by the time the process ends.